### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Estherstarr**
In this PR, I implemented the following:
- [ ] Create a method called current_user in your ApplicationController to make current user data available to all controllers.
     - It will return the first user from the database.

- [ ] Create forms to perform the following functions:
   - Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
   - Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
   - Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -13,13 +13,31 @@
...
+      <p class = "pb-5"><% comment.author.name%> <%=comment.text %></p>
+    <% end %>
 
+    <h4>Add comment</h4>
+    <%= form_with url: comments_path(@post) do |form| %>
+    <div>
+      <%= form.text_area :text %>
+      </div> 
+    <button class = "border border-black rounded px-0.5 py-0.5 "><%= form.submit "Submit" %></button>
+    <% end %>
```
Comment (CollinsTatang) [inline app/views/posts/show.html.erb]: - [ ] Please the should be a form in the `views/comments/new.html.erb `file which accepts input for creating comments.
You can create the comment form separately and render it to the post view.

File: config/routes.rb [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+  resources :users, only: [:index, :show]  do
+    resources :posts, only: [:index, :show] do
+      resources :likes, only: [:create]
+    end
   end
+
+  get 'posts/new', to: 'posts#new'
+  post 'posts', to: 'posts#create'
+  post 'posts/:id/comments', as: 'comments', to: 'comments#create'
+  # post 'posts/:id/likes', as: 'likes', to: 'comments#create'
```
Comment (CollinsTatang) [inline config/routes.rb]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Estherstarr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Estherstarr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**